simulator/bpu/bp_interface.h

```diff
@@ -0,0 +1,21 @@
+/**
+ * bp_interface.h - perfoming BP updates using ports
+ * @author Aleksandr Misevich
+ * Copyright 2018 MIPT-MIPS
+ */
+
+#ifndef BP_INTERFACE_H_
+#define BP_INTERFACE_H_
+
+#include <infra/types.h>
+
+/*the structure of data sent from memory to fetch stage */
+struct BPUpdateInfo{
```

Comment:
That should not contain "Update" as it is used for several purposes. I would call it BPInterface.